packages/app/src/app/app.module.ts

```diff
@@ -1,50 +1,26 @@
 import {HttpModule} from '@nestjs/axios';
-import {MiddlewareConsumer, Module, NestModule, Scope} from '@nestjs/common';
-import {REQUEST} from '@nestjs/core';
+import {ExecutionContext, Module} from '@nestjs/common';
 import {ServeStaticModule} from '@nestjs/serve-static';
-import {AsyncLocalStorageTransactionContext, LoggingHook, OpenFeatureLogger} from '@openfeature/extra';
-import { FlagMetadata, OpenFeature } from '@openfeature/server-sdk';
-import {MetricsHook, TracingHook as SpanEventBasedTracingHook} from '@openfeature/open-telemetry-hooks';
+import {LoggingHook, OpenFeatureLogger} from '@openfeature/extra';
+import {FlagMetadata} from '@openfeature/js-sdk';
```

Comment:
I'm surprised this worked unless we didn't remove the `js-sdk` from thepackage.json. This should be exported from the nest sdk, please use that one.